lib/strip_attributes.rb

```diff
@@ -38,6 +38,14 @@ module StripAttributes
   MULTIBYTE_SUPPORTED  = "\u0020" == " "
 
   def self.strip(record_or_string, options = nil)
+    if options && options[:if] && !record_or_string.send(options[:if].to_sym)
+      return record_or_string
+    end
+
+    if options && options[:unless] && record_or_string.send(options[:unless].to_sym)
+      return record_or_string
+    end
```

Comment:
[PLEASE MOVE] These two guard clauses belong as a part of the `strip_record` method, not here. This method accepts a record or a string. Calling `send` on a passed string doesn't make sense here. 

[PLEASE ADD] Also, Rails conventions allow for a `Proc` to be passed to `:if` and `:unless` options in addition to a symbol representing an instance method. Basically, if the value is a symbol or string, use `send`, but if the value `respond_to?(:call)` and the `arity == 1`, use `call` passing in the record. This would allow for something like:

```ruby
strip_attributes if: Proc.new { |user| user.signup_step > 2 }
```